tests/test_discover.py

```diff
@@ -6,6 +8,7 @@
 
 
 EXPECTED_DISCOVERY_CONTENT = './tests/discovery.json'
+EXPECTED_DISCOVERY_SCHEMA_CONTENT = './tests/catalog-schema.json'
```

Comment:
I prefer 'catalog' rather than 'discovery' in variable and file names. 

...Because it makes me think that the 'discovery.json' is something different to 'catalog.json', which I don't think is true.

Not a big issue. Happy to leave as is if you prefer it that way 🙇

Reply:
Renamed :+1: